<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addSouvDialog</class>
 <widget class="QDialog" name="addSouvDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New Souvenir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="souvNameLE">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter Souvenir Name</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="souvPriceDB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
   <property name="specialValueText">
    <string>Enter Price</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="souvTableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addSouvDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addSouvDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
